include/zephyr/net/phy.h

```diff
@@ -31,17 +31,21 @@ extern "C" {
 /** @brief Ethernet link speeds. */
 enum phy_link_speed {
 	/** 10Base-T Half-Duplex */
-	LINK_HALF_10BASE_T		= BIT(0),
+	LINK_HALF_10BASE_T  	= BIT(0),
```

Comment:
Looks like unnecessary change, why do we need to change the white space here?

Reply:
Removed.